tools/integration_tests/emulator_tests/README.md

```diff
@@ -0,0 +1,35 @@
+# Emulator based tests
+## go-proxy server
+Proxy server, which intercepts [storage-testbench](https://github.com/googleapis/storage-testbench) server and perform pre-defined
+retry test.
+
+### Steps to run the test manually
+1. Run storage-testbench server by following [this](https://github.com/googleapis/storage-testbench/tree/main?tab=readme-ov-file#initial-set-up) steps.
+2. Create test-bucket on server with below command.
```

Comment:
Could you please be explicit about the working directory? I mean, from which directory these commands need to be executed.

Reply:
The directory doesn't matter in this case because the server is running on port 9000. We can create a bucket on it by executing this command from any directory.